charts/incubator/palworld/values.yaml

```diff
@@ -267,17 +276,23 @@ workload:
           stdin: true
           probes:
             liveness:
-              enabled: false
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - pgrep -f "{{ .Values.persistence.serverfiles.targetSelector.main.main.mountPath }}/Pal/Binaries/Linux/PalServer-Linux-Test" >/dev/null || exit 1
```

Comment:
pgrep-ing palserver from another pod-container? :D what is this sorcery?

Reply:
im gonna assume its not possible then....